constellation_ticketbox/models.py

```diff
@@ -4,16 +4,18 @@
 
 from guardian.shortcuts import assign_perm, remove_perm, get_perms
 
+
 class Reply(models.Model):
     ticket = models.ForeignKey('Ticket', blank=True, null=True)
     owner = models.ForeignKey(User)
     author = models.CharField(max_length=128, default='Anonymous')
     body = models.TextField(blank=False, null=False)
     timestamp = models.DateTimeField(auto_now_add=True)
-    
+
+
 class Ticket(models.Model):
     STATUS_CHOICES = (
```

Comment:
Status is currently used to determine whether or not a ticket should be archived upon status update. 

Should this be changed to require an explicit archiving when a ticket status is updated, and allow users to enter anything into the status field?